Connectors/Rhino/Speckle.Connectors.RhinoShared/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -242,6 +246,16 @@ CancellationToken cancellationToken
     return Task.FromResult(new HostObjectBuilderResult(bakedObjectIds, conversionResults));
   }
 
+  private string ReplaceInvalidChars(string input)
+  {
```

Comment:
can you include all invalid characters here, and move this to a general helper function to clean all layer names and block definition names. This currently only fixes the base layer name